src/index.js

```diff
@@ -67,7 +67,7 @@ function applyRef (e, ref) {
 function ensureAttrs (objs) {
   const { attrs, events, ref, key, ...props } = objs || {};
   const newRef = ensureRef({ attrs, events, props, ref });
-  return { ref: newRef, ...(key ? { key } : {}) }
```

Comment:
I think it is. Doesn't mean all implementations are, but I would hope that if a value is `undefined` the underlying implementation wouldn't render it. That'd be weird.